### PR TITLE
Bratseth/remove unused rpc server take 2

### PR DESCRIPTION
**Pull request by bratseth**
@hakonhall or @gjoranv please review the second commit here

### CODE REVIEW

**Comment by bratseth** [comment]
I'll merge this now to get a CD run.

**Comment by hakonhall** [comment]
👍

File: container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java [hunk 1]
```diff
@@ -116,6 +127,20 @@ public ConfiguredApplication(ContainerActivator activator,
...
+    }
+
+    /** The container has no rpc methods, but we still need to register it in Slobrok to enable orchestration */
+    private Register registerInSlobrok(SlobroksConfig slobrokConfig, QrConfig qrConfig) {
+        SlobrokList slobrokList = new SlobrokList();
+        slobrokList.setup(slobrokConfig.slobrok().stream().map(SlobroksConfig.Slobrok::connectionspec).toArray(String[]::new));
+        Spec mySpec = new Spec(HostName.getLocalhost(), qrConfig.rpc().port());
+        Register slobrokRegistrator = new Register(new Supervisor(new Transport()), slobrokList, mySpec);
+        slobrokRegistrator.registerName(qrConfig.rpc().slobrokId());
+        log.log(LogLevel.INFO,
```
Comment (gjoranv) [inline container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java]: I assume this method is a NOP when there are no slobroks, except for the log message?

Comment (bratseth) [reply container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java]: I assume so too, but why do you ask? I think we only have two cases: Standalone, where the model will not enable rpc (as far as I can tell) and so not make it to this code, and clustered, where there are always slobroks.

Comment (gjoranv) [reply container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java]: I forgot for a moment that there is a separate StandaloneContainerApplication.

**Pull request merged**